resources/js/Components/ItemIdSearchTextarea.vue

```diff
@@ -105,6 +109,10 @@ defineExpose({validate, serializeInput, validationError});
         .cdx-progress-bar {
             width: 50%;
             margin: auto;
+
+            @media (max-width: $width-breakpoint-tablet) {
+                width: 80%;
```

Comment:
works nicely, i got a cute little progress bar on mobile. 
In this case too we will have to redo this style once tokens are in place